.github/workflows/qa.yml

```diff
@@ -0,0 +1,18 @@
+name: Check for Sign Off
+
+on:
+  workflow_call:
+    inputs:
+      github-token:
+        required: true
+        type: string
+
+jobs:
+  sign-off-check:
+    name: Check sign-off
+    runs-on: ubuntu-latest
+    steps:
+      - name: sign off
+        uses: krogertechnology/sign-off@v1.0.1
+        with:
+          github-token: ${{ inputs.github-token }}
```

Comment:
This would become `secrets.github-token`

Reply:
Updated